lib/ansible/modules/network/aci/aci_interface_policy_leaf_policy_group.py

```diff
@@ -0,0 +1,395 @@
+   aliases: [ link_aggregation_type_name ]
+ link_level_policy:
+   description:
+   - Choice of link_level_policy to be used as part of the leaf policy group to be created.
+   aliases: [ link_level_policy_name ]
+ cdp_policy:
+   description:
+   - Choice of cdp_policy to be used as part of the leaf policy group to be created.
+   aliases: [ cdp_policy_name ]
+ mcp_policy:
+   description:
+   - Choice of mcp_policy to be used as part of the leaf policy group to be created.
+   aliases: [ mcp_policy_name ]
+ lldp_policy:
+   description:
+   - Choice of lldp_policy to be used as part of the leaf policy group to be created.
+   aliases: [ lldp_policy_name ]
+ stp_interface_policy:
+   description:
+   - Choice of stp_interface_policy to be used as part of the leaf policy group to be created.
+   aliases: [ stp_interface_policy_name ]
+ egress_data_plane_policing_policy:
+   description:
+   - Choice of egress_data_plane_policing_policy to be used as part of the leaf policy group to be created.
+   aliases: [ egress_data_plane_policing_policy_name ]
+ ingress_data_plane_policing_policy:
+   description:
+   - Choice of ingress_data_plane_policing_policy to be used as part of the leaf policy group to be created.
+   aliases: [ ingress_data_plane_policing_policy_name ]
+ priority_flow_control_policy:
+   description:
+   - Choice of priority_flow_control_policy to be used as part of the leaf policy group to be created.
+   aliases: [ priority_flow_control_policy_name ]
+ fibre_channel_interface_policy:
+   description:
+   - Choice of fibre_channel_interface_policy to be used as part of the leaf policy group to be created.
+   aliases: [ fibre_channel_interface_policy_name ]
+ slow_drain_policy:
+   description:
+   - Choice of slow_drain_policy to be used as part of the leaf policy group to be created.
+   aliases: [ slow_drain_policy_name ]
+ port_channel_policy:
+   description:
+   - Choice of port_channel_policy to be used as part of the leaf policy group to be created.
+   aliases: [ port_channel_policy_name ]
+ monitoring_policy:
+   description:
+   - Choice of monitoring_policy to be used as part of the leaf policy group to be created.
+   aliases: [ monitoring_policy_name ]
+ storm_control_interface_policy:
+   description:
+   - Choice of storm_control_interface_policy to be used as part of the leaf policy group to be created.
+   aliases: [ storm_control_interface_policy_name ]
+ l2_interface_policy:
+   description:
+   - Choice of l2_interface_policy to be used as part of the leaf policy group to be created.
+   aliases: [ l2_interface_policy_name ]
+ port_security_policy:
+   description:
+   - Choice of port_security_policy to be used as part of the leaf policy group to be created.
+   aliases: [ port_security_policy_name ]
+ attached_entity_profile:
+   description:
+   - Choice of attached_entity_profile (AEP) to be used as part of the leaf policy group to be created.
+   aliases: [ attached_entity_profile_name, AEP, AEP_name ]
```

Comment:
We use `aep` as the default parameter in other modules and we never used `AEP`, `AEP_name` or `attached_entity_profile`. So please use the same parameter name as the **aci_aep** module !